doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -222,43 +227,43 @@ Bluetooth samples
 
 * :ref:`fast_pair_input_device` sample:
 
-    * Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
+  * Added support for the :ref:`ug_nrf54l15_gs` board.
```

Comment:
```suggestion
  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`.
```
Same applies for all the below suggestions. 
Some more places need this updated. For example : https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-14864/nrf/releases_and_maturity/releases/release-notes-changelog.html#enhanced-shockburst-esb

Reply:
Yeah, noticed that one after the doc rendered, thanks Divya :)